.pre-commit-config.yaml

```diff
@@ -29,3 +29,11 @@ repos:
     rev: v1.19.0
     hooks:
     -   id: pyupgrade
+-   repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v0.701
+    hooks:
+    -   id: mypy
+    -   id: mypy
+        args:
+        -   --py2
```

Comment:
A lot of the ignores/hacks were to support py2. Do we care about running mypy under py2?

Reply:
no, we do not

Reply:
wonderful